style/app.scss

```diff
@@ -233,10 +233,51 @@ a:active
   color           : #4c004c;
 }
 
-#disclaimerDiv
+.popup-banner
 {
-  padding   : 0px 1em 0px 1em;
-  font-size : 11pt;
+  font-size        : 11pt;
+  position         : absolute;
+  background-color : #ffdd61;
+  padding          : 5px 15px 5px 15px;
+  z-index          : 1;
+}
+
+.popup-banner.dismiss
+{
+  transition : visibility 0s 0.5s, opacity 0.5s linear;
+  visibility : hidden;
+  opacity    : 0;
+}
+
+.popup-banner a
+{
+  color : #af16af;
+}
+
+.popup-banner label
+{
+  font-weight : lighter;
+  color       : #696969;
+  position    : absolute;
```

Comment:
Don't overuse absolute positioning. In this particular case, you shouldn't assume that there will be empty space in the bottom-right corner of the banner, as this depends on screen size. Use a different layout here. For example, put the "X" in the top-right corner, and the checkbox at the bottom.